packages/shared/styles/helpers/_breakpoints.scss

```diff
@@ -7,7 +7,7 @@ $desktop-xl-min: 1440px;
 $desktop-xxl-min: 1920px;
 // Media mixins
 @mixin for-mobile {
-  @media (max-width: $desktop-min) {
+  @media (max-width: $tablet-max) {
```

Comment:
@justyna-13 now we have just style for mobile/desktop without tablet. IMO we should use `$desktop-min - 1px` to set 1023px here. Please check other places where we use 1024px for mobile and change it to 1023px 👍 check `window.matchMedia("(max-width: 1024px)")`